src/cpuid-templates/src/lib.rs

```diff
@@ -1,1777 +1,3675 @@
 // Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Could you provide more context in the commit message of commit e2484b1e71e815bd8db6943f4a3eeeafba30c67f for future trackability (why and what is difference from the previous one)?

Reply:
Have done, please see https://github.com/firecracker-microvm/firecracker/pull/3400#discussion_r1090589287